crypto/api/v1/api.go

```diff
@@ -119,3 +157,71 @@ func (w *Wrapper) SignJws(ctx context.Context, request SignJwsRequestObject) (Si
 
 	return SignJws200TextResponse(sig), nil
 }
+
+// EncryptJwe handles api calls for encrypting JWE messages
+func (w *Wrapper) EncryptJwe(ctx context.Context, request EncryptJweRequestObject) (EncryptJweResponseObject, error) {
+	encryptRequest := request.Body
+	if err := encryptRequest.validate(); err != nil {
+		return nil, core.InvalidInputError("invalid encrypt request: %w", err)
+	}
+	receiver := encryptRequest.Receiver
+	id, err := did.ParseDIDURL(receiver)
+	if err != nil {
+		return nil, err
+	}
+	var key crypt.PublicKey
+	var keyID ssi.URI
+	if id.DID.IsURL() {
```

Comment:
Perhaps extract this key fetching code into its own helper function?

Reply:
Created helper function.